### ISSUE TITLE
List all languages for each project

### ISSUE THREAD

**Issue by jfacorro** [inaka/inaka.github.io#11]
https://developer.github.com/v3/repos/#list-languages

**Comment by WEREMSOFT** [comment]
@jfacorro  Need more details about this. Please check the site and tell me if this issue still makes sense.

**Comment by jfacorro** [comment]
It's been a long time, but I think this issue was about the projects that use more than one language and that it would make sense to list them all, or maybe just the two that are used the most (which can be figured out with the information returned by the endpoint listed in the description).

**Comment by elbrujohalcon** [comment]
Yeah… the previous site had a filter by language feature and for that to work properly we needed to account for all the languages used in each repo, otherwise, a repo with 51% js / 49% ruby won't be found when looking for "ruby" repos.
I'm not sure if this is something we still need in the current site, tho.
